library/Vanilla/Site/DefaultSiteSection.php

```diff
@@ -123,4 +123,11 @@ public function setApplication(string $app, bool $enable = true) {
     public function getAttributes(): array {
         return [];
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getSectionThemeID() {
+        // TODO: Implement getSectionThemeID() method.
+    }
```

Comment:
We should have this return null I think. Also add it to the interface declaration up top.